cmd/kube-proxy/app/server_test.go

```diff
@@ -502,35 +509,6 @@ kind: KubeProxyConfiguration
 	}
 }
 
-type fakeProxyServerLongRun struct{}
```

Comment:
nit: removed obsolete code, while on the file.

Reply:
Not sure why this code is removed, seems unrelated.

Reply:
It was not being used anywhere, thought of cleaning it up just for hygiene; but I could revert if it creates destruction.